SDF-to-Coords

```diff
@@ -0,0 +1,27 @@
+import os
+
+import numpy as np
+
+def SDF_to_Coords(filename, num_atom):
+    directory = r'/mnt/d/SDFs/structures/'
+    for filename in os.listdir(directory):
+        if filename.endswith(".sdf"):
+            x = np.asarray(filename)
+            for element in x:
+                col_r1 = [:num_atom+1, 5]
+                b = x[:num_atom+1, :4]
```

Comment:
I get a syntax error for the ' : ' in "col_r1 = [:num_atom+1, 5]", and presumably the same error in the next line. This is the same syntax I see in python tutorials.

Reply:
You are presumably missing the `x` from out front the brackets, *i.e.*, it's supposed to be `x[:num_atom + 1, 5]`